pkg/config/config.go

```diff
@@ -85,6 +85,8 @@ type APISIXConfig struct {
 	// DefaultClusterAdminKey is the admin key for the default cluster.
 	// TODO: Obsolete the plain way to specify admin_key, which is insecure.
 	DefaultClusterAdminKey string `json:"default_cluster_admin_key" yaml:"default_cluster_admin_key"`
+	// DefaultClusterClientTimeout is the request timeout for default cluster client.
+	DefaultClusterClientTimeout types.TimeDuration `json:"default_cluster_client_timeout" yaml:"default_cluster_client_timeout"`
```

Comment:
We will introduce `ApisixClusterConfig` to set these options, so don't add it here.